test/validator_test.exs

```diff
@@ -3,15 +3,16 @@ defmodule ValidatorTest do
   doctest Validator
   import Validator
   import Mock
+  import ExUnit.CaptureIO
 
-  describe("Validator.validate_input") do
-    test "it can validate user input and recurse to get valid input" do
+  describe("Validator.validate_move") do
```

Comment:
Do you need to specify it's the `Validator` this test is for when the file itself already details that?